premium/usage.go

```diff
@@ -575,3 +705,23 @@ func (NoOpUsageClient) IncreaseForTable(_ string, _ uint32) error {
 func (NoOpUsageClient) Close() error {
 	return nil
 }
+func roundDown(x, unit uint32) uint32 {
+	a := (x / unit) * unit
+	b := a + unit
+	if a < b {
+		return a
+	}
+	return b
+}
+
+func roundUp(x, unit uint32) uint32 {
+	a := (x / unit) * unit
+	b := a + unit
+
+	if (x - a) < (b - x) {
+		return a
+	}
+	return b
+}
```

Comment:
```suggestion

func roundDown(x, unit uint32) uint32 {
	return x - (x % unit)
}

func roundUp(x, unit uint32) uint32 {
	if x%unit == 0 {
		return x
	}
	return x + (unit - x%unit)
}
```

I'm not sure why the `roundDown` and `roundUp` functions are so complicate? There also seems to be a bug in `roundUp`: for example, `roundUp(2001, 1000) == 2000`, but I'd expect `3000`